package.json

```diff
@@ -45,10 +45,10 @@
     "rpc-reflector": "^1.3.11"
   },
   "peerDependencies": {
-    "@comapeo/core": "^2.2.0"
+    "@comapeo/core": "^3.0.0"
   },
   "devDependencies": {
-    "@comapeo/core": "2.3.1",
+    "@comapeo/core": "^3.0.0",
```

Comment:
why not use the exact version here?

Reply:
Was unsure about conventions, this was added when I did `npm i --save-dev @comapeo/core@3`, will pin to full v